### DISCUSSION TITLE
🙏 [Question] How to blink an icon? ❌

### DISCUSSION THREAD

**Discussion by azat385** [flet-dev/flet#2382] [Q&A]
### Question

How to make blinking icon or text or any control?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
I think you could use the `opacity` property (set to 0 then back to 1) or the `visible` property (boolean) to achieve this.

**Reply by azat385** [reply]
```python
import flet as ft


def main(page: ft.Page):

    def switch_changed(e):
        text.visible = not switch.value
        page.update()

    switch = ft.Switch(label="Blink", value=False, on_change=switch_changed)
    text = ft.Text("Hello, world!")

    page.add(switch, text)


ft.app(main)
```

How to change the state every second? to use async with a sleepdelay?

**Reply by ndonkoHenri** [reply]
You create have that on a thread. Example [here](https://flet.dev/docs/guides/python/user-controls).
